### PR TITLE
Update URLs

### PR DESCRIPTION
**Pull request by ricardovh**

### CODE REVIEW

**Comment by ricardovh** [comment]
There are still some references to ibissource.org that could not yet be removed (https://ibissource.org/iaf/releases/, https://ibissource.org/contact/, https://ibissource.org/, http://maven.ibissource.org/iaf/, //ibissource.org)

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [![CodeFactor](https://www.codefactor.io/repository/github/ibissource/iaf/badge)](https://www.codefactor.io/repository/github/ibissource/iaf)
 [![total GitHub contributors](https://img.shields.io/github/contributors-anon/ibissource/iaf.svg)](https://github.com/ibissource/iaf/graphs/contributors)
 [![Maven Central](https://img.shields.io/maven-central/v/org.ibissource/ibis-adapterframework-parent.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:org.ibissource)
-[![Latest Snapshot](https://img.shields.io/nexus/public/org.ibissource/ibis-adapterframework-core?label=Latest%20Snapshot&server=https%3A%2F%2Fnexus.ibissource.org%2F)](https://nexus.ibissource.org/#view-repositories;releases~browsestorage)
+[![Latest Snapshot](https://img.shields.io/nexus/public/org.ibissource/ibis-adapterframework-core?label=Latest%20Snapshot&server=https%3A%2F%2Fnexus.frankframework.org%2F)](https://nexus.frankframework.org/#view-repositories;releases~browsestorage)
```
Comment (nielsm5) [inline README.md:10]: https://nexus.frankframework.org/#browse/browse

File: pom.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@
 	<url>http://maven.ibissource.org/iaf/</url>
 
 	<organization>
-		<name>Ibissource.org</name>
+		<name>frankframework.org</name>
 		<url>https://ibissource.org</url>
```
Comment (nielsm5) [inline pom.xml:27]:
```suggestion
		<name>Frank!Framework</name>
		<url>https://frankframework.org</url>
```

**Review by nielsm5 — CHANGES_REQUESTED** [review]

**Comment by nielsm5** [comment]
http://maven.ibissource.org/iaf/ mag gewoon https://frankframework.org worden.

**Review by nielsm5 — APPROVED** [review]

**Review by gvanbrakel — APPROVED** [review]

**Pull request merged**